test/remote/gateways/remote_commerce_hub_test.rb

```diff
@@ -42,6 +42,14 @@ def test_successful_purchase
     assert_equal 'Approved', response.message
   end
 
+  def test_successful_purchase_whit_physical_goods_indicator
+    @options[:physical_goods_indicator] = true
+    response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+    assert_equal 'Approved', response.message
+    assert_equal @options[:physical_goods_indicator], response.params['transactionDetails']['physicalGoodsIndicator']
```

Comment:
💬 Comment;

GIven that you are testing a boolean value you can just use `assert`:

```ruby
assert response.params['transactionDetails']['physicalGoodsIndicator']
```